tools/runner/runner.go

```diff
@@ -117,14 +120,13 @@ func (r *Runner) runTest(config *grpcv1.LoadTest, reporter *TestCaseReporter, do
 	for {
 		loadTest, err := r.loadTestGetter.Get(config.Name, metav1.GetOptions{})
 		if err != nil {
-			reporter.Warning("Failed to poll test %s: %v", name, err)
 			if retries < r.retries {
 				retries++
-				reporter.Info("Scheduling retry %d/%d to poll test", retries, r.retries)
+				reporter.Info("Failed to poll test %s (scheduling retry %d/%d): %v", name, retries, r.retries, err)
 				r.afterInterval()
 				continue
 			}
-			reporter.Error("Aborting test after %d retries to poll test %s: %v", r.retries, name, err)
+			reporter.Error("Error polling the test", "Aborting test after %d retries to poll test %q failed: %v", r.retries, name, err)
```

Comment:
Generally speaking, it is unnecessary to mention the name of the test. I would not add "failed" either.